src/transformers/image_transforms.py

```diff
@@ -223,27 +223,31 @@ def resize(
     image,
     size: Tuple[int, int],
     resample=PILImageResampling.BILINEAR,
+    reducing_gap: Optional[int] = None,
     data_format: Optional[ChannelDimension] = None,
     return_numpy: bool = True,
 ) -> np.ndarray:
     """
-    Resizes `image` to (h, w) specified by `size` using the PIL library.
+    Resizes *image* to (h, w) specified by *size* using the PIL library.
```

Comment:
Just a question for future work, are we switching to using bold formatting instead of italic for arguments and data types?

Reply:
Oh, no, absolutely not. I'll fix this - this is just because I used code pilot and didn't catch this

Reply:
Done!